components/TextImage.vue

```diff
@@ -1,11 +1,13 @@
 <template>
   <div>
-    <article>
-      <div>
-        <h2>{{ blok.title }}</h2>
-        <rich-text-renderer :document="blok.description" />
+    <article class="container xl:flex l:flex">
```

Comment:
l:flex bestaat niet. lg:flex wel. als je lg:flex gebruikt hoeft je niet xl:flex te gebruiken want bij xl pakt die automatisch styling van de kleinere schermen.

Reply:
Voeg een padding aan de onderkant van de he;e component zodat de volgende component die er onder komt niet te dichtbij zit. Je weet niet wat de volgende component is omdat de pagina dynamisch is, hou rekening met dat